### ISSUE TITLE
Errors on Debian 12 (bookworm)

### ISSUE THREAD

**Issue by ARS101** [parchlinuxB/namban#4]
Hi.
Every time I open the program I get these errors repeatedly.

```
(python3:96294): GLib-GIO-CRITICAL **: 16:33:09.328: g_dbus_connection_emit_signal: assertion 'object_path != NULL && g_variant_is_object_path (object_path)' failed
```

And everytime I enable a dns server I get this error:

```
cp: cannot create regular file '/usr/share/namban/oldconffiles/resolv.conf': No such file or directory
Traceback (most recent call last):
  File "/usr/lib/namban/src/view/__init__.py", line 30, in onChange
    self.app.connectProfile(self.domain)
  File "/usr/lib/namban/src/control.py", line 27, in connectProfile
    self.systemDnsSet(profile)
  File "/usr/lib/namban/src/core/__init__.py", line 9, in systemDnsSet
    self.currentStrategy.connect()
  File "/usr/lib/namban/src/core/strategy.py", line 45, in connect
    self.check4resovConfFile()
  File "/usr/lib/namban/src/core/strategy.py", line 32, in check4resovConfFile
    resolvConfPromise((settings.APP_FILES_PATH/'oldconffiles/resolv.conf').__str__())
  File "/usr/lib/namban/src/core/promises.py", line 13, in __init__
    super().__init__(path=path,**d)
  File "/usr/lib/namban/src/core/basePromise.py", line 12, in __init__
    f = open(fp, "w+")
        ^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/usr/share/namban/promises/803f9176faf45d6c2ea6ee2905bdbcd2455381b023488ab2f705a6e7b6aa5be7.promise'
```

OS: Debian 12 (bookworm)
App version: v0.2

**Comment by meshya** [comment]
What about v0.3 ?

**Comment by meshya** [comment]
Fixed in [v0.3-1](https://github.com/parchlinuxB/namban/releases/tag/v0.3-1)
If you see this bug 🐛 again, open a new issue

**Comment by ARS101** [comment]
Can confirm it is fixed.
